### PR TITLE
IMB-204 Internal server error

### PR DESCRIPTION
**Pull request by muthrajan**
## What
500 Internal server error [IMB-204](https://collaboration.homeoffice.gov.uk/jira/browse/IMB-204)

## Why
-  When the user is on the 'Are there any other reasons you should be granted temporary permission to stay in the UK?' page and they click on 'Yes', then click on 'Human Rights' and type something in the text box, click continue and then click continue on the upload evidence page without uploading any documents they receive a 500 internal server error.

## How
- In summary-data-section.js in  temporary-permission-to-stay section typeof(req.sessionModel.get('temporary-permission-reasons')) === 'object' was checked for join and if it is string join was not used

## Testing
- Tests passing locally

### CODE REVIEW

**Review by mislam987 — APPROVED** [review]
Looks okay to me.
